### DISCUSSION TITLE
It just wont work :(

### DISCUSSION THREAD

**Discussion by R4ygen** [proginosko/LeechBlockNG#513] [Q&A] [answered]
Hello, I've installed this extension yesterday to block youtube (15 minutes max every hour), and it worked perfectly! But today, it wont work anymore. Nothing happened between yesterday and today, just turned on and off my pc.
In the statistics, it records the time spent in the website, but the "time left in limit period" won't go down, and there is no timer appearing in the top left in the webpage.

- I've checked all the settings and documentation, nothing seems to be off in the settings (I think, at least)
- I've tried to reboot my pc, again and again.
- I've tried with Firefox in private mode and with all the other extensions disabled, nothing.
- I've tried to uninstall and reinstall the extension (and resetting it, by so), nothing.

I have no idea what's happening.
Only thing I didn't tried is to reinstall Firefox buuut I though about asking here before taking such drastic measures. I hope it's really  just something stupid I'm missing.

I've attached my settings file.
[LeechBlockOptions.txt](https://github.com/user-attachments/files/17967590/LeechBlockOptions.txt)

**Comment by proginosko** [comment, marked as answer]
You currently have it set to block only on Monday-Friday, and today is Saturday. Could that be it?

**Reply by R4ygen** [reply]
That was it. It works.
I knew it was something stupid.
I thought the option "withing time period OR after time limits" it included the days as "time period".
Thank you very much!
